srcpkgs/gdb/template

```diff
@@ -1,65 +1,60 @@
 # Template file for 'gdb'
 pkgname=gdb
 version=10.1
-revision=3
+revision=4
 build_style=gnu-configure
 pycompile_dirs="/usr/share/gdb/python"
 configure_args="--disable-werror --disable-nls --with-system-readline
  --with-system-gdbinit=/etc/gdb/gdbinit --with-system-zlib $(vopt_enable gdbserver)
  $(vopt_if static 'CFLAGS=-static CXXFLAGS=-static LDFLAGS=-static')
- $(vopt_if python --with-python=/usr/bin/python3)
+ $(vopt_with guile) $(vopt_if python --with-python=/usr/bin/python3)
  $(vopt_with debuginfod)"
-hostmakedepends="texinfo perl pkg-config $(vopt_if python python3-devel)"
-makedepends="ncurses-devel zlib-devel readline-devel expat-devel
- $(vopt_if python 'python3-devel gettext-devel')
- $(vopt_if debuginfod elfutils-devel)"
+hostmakedepends="pkg-config texinfo $(vopt_if python python3-devel) $(vopt_if guile guile)"
+makedepends="expat-devel ncurses-devel readline-devel zlib-devel $(vopt_if guile guile-devel)
+ $(vopt_if python 'python3-devel gettext-devel') $(vopt_if debuginfod elfutils-devel)"
+depends="gdb-common-${version}_${revision}"
+checkdepends="dejagnu"
 short_desc="GNU Debugger"
 maintainer="Anthony Iliopoulos <ailiop@altatus.com>"
 license="GPL-3.0-or-later"
-homepage="https://www.gnu.org/software/gdb/"
-distfiles="${GNU_SITE}/${pkgname}/${pkgname}-${version}.tar.xz"
+homepage="https://www.gnu.org/software/gdb"
+distfiles="${GNU_SITE}/gdb/gdb-${version}.tar.xz"
 checksum=f82f1eceeec14a3afa2de8d9b0d3c91d5a3820e23e0a01bbb70ef9f0276b62c0
+make_check=extended  # Tests take too long, not all of them pass.
 python_version=3
 patch_args="-Np1"
+ignore_elf_files="
+ /usr/share/gdb/guile/gdb/support.go
+ /usr/share/gdb/guile/gdb/experimental.go
+ /usr/share/gdb/guile/gdb/iterator.go
+ /usr/share/gdb/guile/gdb/types.go
+ /usr/share/gdb/guile/gdb/printing.go
+ /usr/share/gdb/guile/gdb.go"
 
 if [ "${CROSS_BUILD}" ]; then
 	# Make python3.x detection work in cross builds
 	CFLAGS="-I${XBPS_CROSS_BASE}/${py3_inc}"
 	CXXFLAGS="-I${XBPS_CROSS_BASE}/${py3_inc}"
+	makedepends+=" libatomic_ops-devel" # guile
 fi
-CFLAGS+=" -fcommon"
-CXXFLAGS+=" -fcommon"
+
 # Package build options
-build_options="gdbserver static python debuginfod"
+build_options="debuginfod gdbserver guile python static"
 desc_option_gdbserver="Enable support for building GDB server"
 desc_option_debuginfod="Enable support for libdebuginfod"
 # Enable gdbserver if !static.
 build_options_default="gdbserver python debuginfod"
```

Comment:
Makes sense to reorder these as well?

Reply:
It's just personal preference :) Should I undo the changes?

Reply:
I don't think undoing is necessary; just suggesting keeping `default` in the same order as `build_options`. Alphabetical is as good as any.

Reply:
I don't think undoing is necessary; just suggesting keeping `default` in the same order as `build_options`. Alphabetical is as good as any.

Reply:
> suggesting keeping default in the same order as build_options

Fixed.